packages/nc-gui/utils/urlUtils.ts

```diff
@@ -20,7 +20,7 @@ export const replaceUrlsWithLink = (text: string): boolean | string => {
 }
 
 export const isValidURL = (str: string) => {
-  return isURL(`${str}`)
+  return isURL(`${str}`, protocols = ['http', 'https', 'ftp', 'file'])
```

Comment:
I guess the option(2nd argument) should be an object. 

From docs:
<img width="579" alt="Screenshot 2023-02-09 at 1 29 52 PM" src="https://user-images.githubusercontent.com/61551451/217752068-e3f06605-b948-409b-b5aa-ce84512e4e2c.png">


So it will be 

```js
return isURL(`${str}`, { protocols : ['http', 'https', 'ftp', 'file'] })
```